contracts/BaseToken.sol

```diff
@@ -147,7 +140,11 @@ abstract contract BaseToken is
         address to,
         uint256 firstTokenId,
         uint256 batchSize
-    ) internal virtual override(ERC721Enumerable) {
+    )
+        internal
+        virtual
+        override(ERC721Enumerable)
+    {
```

Comment:
All changes in this file are the result of running foundry's built-in formatter.
No change in logic here.